### ISSUE TITLE
Error building CentOS 5 sending it the debug parameter

### ISSUE THREAD

**Issue by fcaffieri** [wazuh/wazuh-packages#1872]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
|4.4.0| Build package |build.sh CentOS 5| Script |CentOS 5 |

## Description
When running `build.sh` from Jenkins adding the debug parameter, it searches for the script `%{_rpmconfigdir}/find-debuginfo.sh` set in the SPECS, and cannot find it in said directory.
And it throws the following error:

```
19:15:45 + '[' yes = yes ']'
19:15:45 + '%{_rpmconfigdir}/find-debuginfo.sh'
19:15:45 /var/tmp/rpm-tmp.328: line 145: fg: no job control
19:15:45 error: Bad exit status from /var/tmp/rpm-tmp.328 (%install)
19:15:45 RPM build errors:
19:15:45 Bad exit status from /var/tmp/rpm-tmp.328 (%install)
19:15:45
19:15:45 [Pipeline] echo
19:15:45 Package generation ended with exit code: 1
```

## Evidences
Test with Debug:
https://devel.ci.wazuh.info/job/Package_builder_fcaffieri/247

Test without Debug:
https://devel.ci.wazuh.info/job/Package_builder_fcaffieri/249

## Steps to reproduce

Execute this pipeline [Package_builder](https://devel.ci.wazuh.info/job/Package_builder_fcaffieri/) with the following parameters:

![image](https://user-images.githubusercontent.com/89791732/193697109-17d4bb63-8ae4-49c6-8eaa-c0863034d15b.png)
![image](https://user-images.githubusercontent.com/89791732/193697159-0faf7be5-d156-4c8c-af4e-5d4a0f26f2c9.png)

**Comment by lchico** [comment]
# Update

This will be fixed in issue  https://github.com/wazuh/wazuh/issues/21755, wich will be merged in **release 4.9.0**, and it has been tested on [this workflow](https://github.com/wazuh/wazuh/actions/runs/8726546151/job/23942073546).

**Comment by MarcelKemp** [comment]
I close the issue because it will be solved when we merge the next PR:
- https://github.com/wazuh/wazuh/pull/22396
